### PR TITLE
NSFS | NC | move /etc/noobaa.conf.d/.env location to /etc/sysconfig/noobaa and redirect file read addition

### PR DESCRIPTION
**Pull request by romayalon**
### Explain the changes
This is the first part of the new configuration - 
1. nsfs.js - read /etc/noobaa.conf.d/config_dir_redirect and update the path appears in this file as config_root.
2. nsfs.service - replace .env location - /etc/noobaa.conf.d/.env with -/etc/sysconfig/noobaa
3. updated docs

### Issues: Fixed #xxx / Gap #xxx
1. https://issues.redhat.com/browse/MCGI-212
2. https://issues.redhat.com/browse/MCGI-217

### Testing Instructions:
1. Upgrade was tested manually 

- [x] Doc added/updated
- [ ] Tests added

### CODE REVIEW

File: docs/non_containerized_NSFS.md [hunk 1]
```diff
@@ -228,7 +244,7 @@ Health status of the NSFS can be fetched using the command line.
  ```
 
 ## Bucket and Account Manage CLI
-Users can create, update, delete, and list buckets and accounts using CLI. If the config directory is missing CLI will create one and also create accounts and buckets sub-directories in it and default config directory is `/etc/noobaa.conf.d`. 
+Users can create, update, delete, and list buckets and accounts using CLI. If the config directory is missing CLI will create one and also create accounts and buckets sub-directories in it and default config directory is `${config.NSFS_NC_NOOBAA_CONF_DIR}`.
```
Comment (naveenpaul1) [inline docs/non_containerized_NSFS.md:247]: is its `${config.NSFS_NC_DEFAULT_CONF_DIR}` instead of `${config.NSFS_NC_NOOBAA_CONF_DIR}` ?

Comment (romayalon) [reply docs/non_containerized_NSFS.md]: thanks

File: src/deploy/noobaa_nsfs.service [hunk 2]
```diff
@@ -8,7 +8,7 @@ RestartSec=2
 User=root
 Group=root
 ExecStart=/usr/local/noobaa-core/bin/node /usr/local/noobaa-core/src/cmd/nsfs.js
-EnvironmentFile=/etc/noobaa.conf.d/.env
+EnvironmentFile=-/etc/sysconfig/noobaa_nsfs
```
Comment (naveenpaul1) [inline src/deploy/noobaa_nsfs.service:11]: This env file is mandatory for system or with `-` it will be optional?

**Review by naveenpaul1 — APPROVED** [review]
Small comment, 
LGTM

File: src/deploy/noobaa_nsfs.service (hunk 2 above)
Comment (romayalon) [reply src/deploy/noobaa_nsfs.service:11]: yes exactly, -path makes the path optional

**Pull request merged**